tests/util/test_money.py

```diff
@@ -0,0 +1,75 @@
+"""Test the Money class."""
+import pytest
+
+from coin_test.util import Money, Ticker
+
+
+def test_initialization() -> None:
+    """Initialize correctly."""
+    example_ticker = Ticker("BTC")
+    example_qty = 12.5
+
+    Money(example_ticker, example_qty)
```

Comment:
should there be an assert here for the ticker and the qty

Reply:
Adding that now